hooks/scene_operation_tk-shell.py

```diff
@@ -68,4 +68,7 @@ def execute(self, operation, file_path, context, parent_action, file_version, re
             return os.getcwd()
         elif operation == "reset":
             return True
-
+        elif operation == "open":
+            return QtGui.QMessageBox.question(
+                None, "", "Are you sure?", QtGui.QMessageBox.Yes | QtGui.QMessageBox.No
```

Comment:
Are we sure of what? If this message makes sense in context, then it's fine. Just making sure...